plugin/src/main/resources/io/jenkins/plugins/analysis/core/model/ResultAction/summary.jelly

```diff
@@ -1,130 +1,132 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:t="/lib/hudson" xmlns:st="jelly:stapler"
-         xmlns:fa="/font-awesome" xmlns:c="/controls">
+         xmlns:fa="/font-awesome" xmlns:c="/controls" xmlns:l="/lib/layout">
+  <l:userExperimentalFlag var="newBuildPage" flagClassName="jenkins.model.experimentalflags.NewBuildPageUserExperimentalFlag" />
 
-  <st:once>
-    <link rel="stylesheet" href="${resURL}/plugin/warnings-ng/css/custom-style.css"/>
-    <script type="text/javascript" src="${resURL}/plugin/warnings-ng/js/reset-quality-gate.js"/>
-  </st:once>
+  <j:if test="${!newBuildPage}">
```

Comment:
Are there actually changes in this file? Or is Git creating this diff only due to the new indention for the `newBuildPage` guard?